python/lsst/afw/table/match.cc

```diff
@@ -22,16 +22,78 @@
 
 #include <pybind11/pybind11.h>
 //#include <pybind11/operators.h>
-//#include <pybind11/stl.h>
+#include <pybind11/stl.h>
+
+#include "lsst/pex/config/pybind11.h"  // for LSST_DECLARE_CONTROL_FIELD
```

Comment:
Thank you for this comment. That will make it much more useful to find out where this is declared in the future.